src/vmm/src/lib.rs

```diff
@@ -42,7 +42,7 @@ pub mod builder;
 /// Syscalls allowed through the seccomp filter.
 pub mod default_syscalls;
 pub(crate) mod device_manager;
-pub(crate) mod memory_dump;
+pub mod memory_dump;
```

Comment:
We could find a better name for this module, since now it can do more than dumping memory. Maybe `memory_snapshot` ?

Reply:
Done.